apps/firelens-stability/collections/ecs-firelens-stability-tests/s3-stability/cases/s3-throughput-30mbps.json

```diff
@@ -0,0 +1,5 @@
+{
+    "definitions": {
+        "datajet_kbps_throughput_four_tcp": "7500"
```

Comment:
This is the amount of data sent to fluent bit. The actual data stored in s3 may be more as it includes ECS metadata like the following:
```
{"date":"2023-04-12T19:20:59.298693Z","log":"{\"log\":\"<1kb of data here>\"}","ecs_cluster":"DEMO-2_28_5-stability-tests-r3-E-2023-04-12T191930525-iTZ4MHNx-ecs-firelens-stability-tests","ecs_task_arn":"arn:aws:ecs:us-west-2:210752607241:task/DEMO-2_28_5-stability-tests-r3-E-2023-04-12T191930525-iTZ4MHNx-ecs-firelens-stability-tests/79b9e0854bbb4917a7a466d1aaea7f23","ecs_task_definition":"ecs-firelens-stability-tests-s3-sensitivity-s3-s3_compression-gzip:10"}
```